PyPrograms/Raspberry Pi Detector/main.py

```diff
@@ -0,0 +1,12 @@
+# Raspberry Pi Detector
+# By @bupboi1337
+
+import os
+hostname = "raspberrypi.local" # Raspberry Pi Local Hostname
```

Comment:
I could have made hostname just host and make response just ping for a slightly smaller filesize